src/Microsoft.AspNet.Http.Abstractions/Authentication/AuthenticationManager.cs

```diff
@@ -11,6 +11,11 @@ namespace Microsoft.AspNet.Http.Authentication
 {
     public abstract class AuthenticationManager
     {
+        /// <summary>
+        /// Constant used to represent the automatic scheme
+        /// </summary>
+        public const string AutomaticScheme = "(_automatic_)";
```

Comment:
That will look terrible in error messages. `Automatic` should be fine, I'm not too worried about collisions.